### ISSUE TITLE
Errors not useful

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/name-your-contributors#7]
I got the following error. It really doesn't help me; somewhere, something failed to paginate. What we should specify is in what dependency, where, and how many hits we have had to the API - maybe that is why we get `server error` issue. 

```
$ name-your-contributors ipfs --since=2016-02-29T12:00:01Z 
Failed to depaginate
 { [Error: Error: connect ENETUNREACH 192.30.252.127:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ENETUNREACH 192.30.252.127:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: {"message":"Server Error","documentation_url":"https://developer.github.com/v3"}]
  status: 500,
  json: 
   { message: 'Server Error',
     documentation_url: 'https://developer.github.com/v3' } }
Failed to depaginate
 { [Error: {"message":"Server Error","documentation_url":"https://developer.github.com/v3"}]
  status: 500,
  json: 
   { message: 'Server Error',
     documentation_url: 'https://developer.github.com/v3' } }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
```

**Comment by RichardLitt** [comment]
Still getting this. Looks like some dep doesn't have the new depag. 

```
🐕  node cli.js ipfs --since=2016-04-13T12:00:01Z --until=2016-04-25T12:00:01Z
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: connect ECONNREFUSED 192.30.252.126:443
    at Object.exports._errnoException (util.js:860:11)
    at exports._exceptionWithHostPort (util.js:883:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1063:14)] status: 0 }
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
Failed to depaginate!
 { [Error: {"message":"Server Error","documentation_url":"https://developer.github.com/v3"}]
  status: 500,
  json: 
   { message: 'Server Error',
     documentation_url: 'https://developer.github.com/v3' } }
Options: { org: 'ipfs',
  repoName: 'js-ipfs-api',
  since: '2016-04-13T12:00:01Z',
  state: 'all',
  page: 2 }
Trace
    at /Users/richard/src/name-your-contributors/node_modules/depaginate/index.js:26:17
    at tryCatcher (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/promise.js:502:31)
    at Promise._settlePromise (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/promise.js:559:18)
    at Promise._settlePromise0 (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/promise.js:604:10)
    at Promise._settlePromises (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/promise.js:679:18)
    at Async._drainQueue (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/async.js:138:16)
    at Async._drainQueues (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/async.js:148:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/richard/src/name-your-contributors/node_modules/bluebird/js/release/async.js:17:14)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
err [TypeError: Cannot read property 'updatedAt' of undefined]
Failed to depaginate
 { [Error: Error: socket hang up
    at createHangUpError (_http_client.js:209:15)
    at TLSSocket.socketOnEnd (_http_client.js:294:23)
    at emitNone (events.js:72:20)
    at TLSSocket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:905:12)
    at doNTCallback2 (node.js:450:9)
    at process._tickCallback (node.js:364:17)] status: 0 }
err [TypeError: Cannot read property 'updatedAt' of undefined]
```

**Comment by RichardLitt** [comment]
Should be fixed with 1.1.0
